Pipfile

```diff
@@ -0,0 +1,20 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+setuptools = "==69.5.1"
+wheel = "==0.43.0"
+numpy = "==1.26.4"
+astropy = "==6.0.1"
+
+[dev-packages]
+flake8 = "==7.0.0"
+black = "==24.4.2"
+pre-commit = "==3.7.0"
+pytest = "==8.2.0"
+ipython = "==8.24.0"
+
+[requires]
+python_version = "3.12"
```

Comment:
Not a lot of people, especially in astrophysics, are using Python 3.12 at the moment. E.g. on Typhon we only have 3.6.8. I would recommend pinning to the lower version so that, again, people will have an easy time installing.

Reply:
Would you pin as low as 3.6?

Reply:
I have very strong opinions founded on no basis than 3.8 is the lowest you should go, and force people to update. Also
![Thu  2 May 17:18:24 BST 2024](https://github.com/hstewart93/continunet/assets/11492844/3b9e018a-5b80-4e26-bdb0-4d4e8a8b0e5a)

The [Python release cycle](https://devguide.python.org/versions/) has deprecated anything before 3.8.